spacepy/time.py

```diff
@@ -131,19 +131,31 @@ class Ticktock(MutableSequence):
     (TAI, UTC, ISO, JD, MJD, UNX, RDT, CDF, DOY, eDOY)
 
     Possible input data types:
-    ISO: ISO standard format like '2002-02-25T12:20:30'
-    UTC: datetime object with UTC time
-    TAI: elapsed seconds since 1958/1/1 (includes leap seconds)
-    UNX: elapsed seconds since 1970/1/1 (all days have 86400 secs sometimes unequal lenghts)
-    JD: Julian days elapsed
-    MJD: Modified Julian days
-    RDT: Rata Die days elapsed since 1/1/1
-    CDF: CDF epoch: milliseconds since 1/1/0000
-
-    Possible output data types:
-    All those listed above, plus
-    DOY:  Integer day of year, starts with day 1
-    eDOY: Fractional day of year, starts at day 0
+
+    ISO
+        ISO standard format like '2002-02-25T12:20:30'
+    UTC
+        datetime object with UTC time
+    TAI
+        elapsed seconds since 1958/1/1 (includes leap seconds)
```

Comment:
Elapsed should be capitalized for consistency with the other entries

Reply:
Good point. Force-pushed updated commit (seemed not worth having another commit in history just for this.)